Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM python:3.8
+ARG PYTHON_VERSION=3.8
```

Comment:
Good idea to have a default argument here 👍 If it was another Dockerfile (e.g. one that is only used in the workflow), then it would be better to leave it out